packages/codegen-sdk/src/print-model.ts

```diff
@@ -249,7 +249,7 @@ function printModel(context: SdkPluginContext, model: SdkModel): string {
             return printLines([
               printComment([field.description?.value]),
               printDebug(operation),
-              `public ${getMutationName(model, operation.name)}(${printList([mutationArgs.printInput]) ?? ""}) {
+              `public ${getMutationTypeFromName(model, operation.key)}(${printList([mutationArgs.printInput]) ?? ""}) {
```

Comment:
Why `key` instead of `name`? Feels at odds with the rename of the function to `...FromName`

Reply:
I initially renamed getMutationName to getMutationTypeFromName to better reflect the fact that this method was actually getting a mutation type rather than a name. 
To alleviate any further confusion, I will update naming here.